examples/model_hosting/simple_train_predict/linreg/linreg/model.py

```diff
@@ -37,9 +38,12 @@ def train(file_io, data_spec, api_key, project, **kwargs):
             used to initiate this training routine through the Model Hosting
```

Comment:
Maybe we don't need to mention this anymore - and only have it in the technical documentation for the interested reader?